bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/heating.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="evcc"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<thing-type id="heating" listed="false">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+
+		<label>Heating - Loadpoint</label>
+		<description>A heating loadpoint configured in your evcc instance</description>
+	</thing-type>
+	<!-- This thing is currently using the chame channels as a loadpoint does -->
```

Comment:
Applies to the other Thing's too:
Probably see it later on, but i guess these channels are added automatically, The same for configuration and or properties? Are you able to preset a representation property in the xml?